tests/unit_tests/Commands/Plan/InfoCommandTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Plan;
+
+use Consolidation\OutputFormatters\StructuredData\PropertyList;
+use Pantheon\Terminus\Commands\Plan\InfoCommand;
+use Pantheon\Terminus\Models\Plan;
+use Pantheon\Terminus\UnitTests\Commands\CommandTestCase;
+
+/**
+ * Class InfoCommandTest
+ * Test suite class for Pantheon\Terminus\Commands\Plan\InfoCommand
+ * @package Pantheon\Terminus\UnitTests\Commands\Plan
+ */
+class InfoCommandTest extends CommandTestCase
+{
+    /**
+     * @var Plan
+     */
+    protected $plan;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setup()
+    {
+        parent::setUp();
+
+        $this->plan = $this->getMockBuilder(Plan::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->command = new InfoCommand($this->getConfig());
+        $this->command->setSites($this->sites);
+        $this->command->setLogger($this->logger);
+    }
+
+    /**
+     * Exercises plan:info
+     */
+    public function testInfo()
+    {
+        $data = [
+            'billing_cycle' => 'monthly',
+            'id' => 'plan id',
+            'name' => 'Plan A',
+            'price' => 2599,
+            'monthly_price' => 2599,
+            'sku' => 'Plan.sku',
+            'storage' => 1234,
+            'support_plan' => 'silver',
+        ];
+
+        $this->site->expects($this->once())
```

Comment:
This test does not actually make any meaningful assertions. In order for this test to add value to the implementation, it would need to make one final assertion confirming that the data injected through the mocks was returned in some meaningful way in the final result (the PropertyList).

Example Annotated Command integration tests: 

- https://github.com/consolidation/annotated-command/blob/master/tests/FullStackTest.php#L85
- https://github.com/consolidation/annotated-command/blob/master/tests/AnnotatedCommandFactoryTest.php